### ISSUE TITLE
Show inscription field if inscription tag is present for any object

### ISSUE THREAD

**Issue by matkoniecz** [openstreetmap/iD#8157]
Basically anything may have inscription, so inscription tag can be present on nearly anything. If there is sane way to always display it of tagged already it would be nice to do this.

https://www.openstreetmap.org/node/1782223379
![screen03](https://user-images.githubusercontent.com/899988/97970782-5a40f700-1dc2-11eb-8e0d-7d7637940ac0.png)

**Comment by quincylvania** [comment]
@matkoniecz It might be okay to add this field to more features, but I don't think we can show this for everything since an inscription doesn't make sense for many types of OSM features: rivers, lakes, wetlands, woods, turn restrictions, destinations, sites, transit stop areas, boundaries, ferry routes…

**Comment by matkoniecz** [comment]
> rivers, lakes, wetlands, woods

Good point, what I proposed was clearly too extreme.

**Comment by sun-geo** [comment]
I agree, I think we shouldn't add the field `inscription` everywhere, same as i think we don't need the field `name` for all features, see #8109 ;-)

**Comment by matkoniecz** [comment]
Looking at it again, important part of my proposal is to do it in cases where `inscription` tag is added already

So if river or forest is tagged with it tagging does not really make sense, but it would be useful to show it.

**Comment by MapSpot** [comment]
Coming back to an old thread. The four iD presets from my experience that noticeably lack an inscription field are Statue, Sculpture, Picnic Table, and Drinking Water. These would definitely be a good start to pursue.

**Comment by tordans** [comment]
@MapSpot this would (now) be something for the id-tagging-schema repo.
